app/tests/test_favourite_view.py

```diff
@@ -0,0 +1,45 @@
+from django.test import TestCase
+from app.models import Tender
+from django.contrib.auth.models import User
+
+
+def create_tender(status):
+    return Tender.objects.create(
+        title='Tender3',
+        reference='RFC/TEST/1237',
+        url='http://test.com',
+        source='UNGM',
+        unspsc_codes='98765',
+        favourite=True,
+    )
+
+
+def get_url(tender_id):
+    return '/app/tenders/favourite/' + str(tender_id) + '/'
```

Comment:
Change to reverse https://stackoverflow.com/questions/659832/reversing-django-urls-with-extra-options